### PR TITLE
FF119RelNote: WebTransport supports sendorder

### PR DESCRIPTION
**Pull request by hamishwillee**
FF119 supports `options.sendOrder` as an option to the `WebTransport.createBidirectionalStream()` and  `WebTransport.createUnidirectionalStream()` in https://bugzilla.mozilla.org/show_bug.cgi?id=1816925

This adds a  release note. 

Related docs work can be tracked in https://github.com/mdn/content/issues/29300

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/119/index.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ This article provides information about the changes in Firefox 119 that affect d
 
 ### APIs
 
+- The relative priority for send streams can now be specified in an options argument passed to [`WebTransport.createBidirectionalStream()`](/en-US/docs/Web/API/WebTransport/createBidirectionalStream) and [`WebTransport.createUnidirectionalStream()`](/en-US/docs/Web/API/WebTransport/createUnidirectionalStream) ([Firefox bug 1816925](https://bugzil.la/1816925)).
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/firefox/releases/119/index.md:43]: Would be good to mention the name of the property. How about:

```suggestion
- The relative priority for send streams can now be specified by including the `sendOrder` property inside an options argument passed to [`WebTransport.createBidirectionalStream()`](/en-US/docs/Web/API/WebTransport/createBidirectionalStream) and [`WebTransport.createUnidirectionalStream()`](/en-US/docs/Web/API/WebTransport/createUnidirectionalStream) ([Firefox bug 1816925](https://bugzil.la/1816925)).
```

**Review by chrisdavidmills — APPROVED** [review]
@hamishwillee approving, with a small suggestion.

We should also document `sendOrder`, asap. Are you working on that somewhere else?

**Comment by hamishwillee** [comment]
> We should also document sendOrder, asap. Are you working on that somewhere else?

Yes, that's waiting for review in https://github.com/mdn/content/pull/29359

File: files/en-us/mozilla/firefox/releases/119/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/119/index.md:43]: Sounds better to me. Thank you very much.

**Pull request merged**